src/test/pmemblk_create/README

```diff
@@ -0,0 +1,11 @@
+Linux NVM Library
+
+This is src/test/pmemblk_create/README.
+
+This directory contains a unit test for pmemblk_create().
```

Comment:
Yes, I agree, good idea.  Please see the modified pmemblk_create() test program.  Please let me know if this meets with your approval.  And on a more general note regarding the error code returned by the pmemblk_open (et. al.) entry point, there are a number of code paths that may be taken inside the API routine itself that all result in the same thing being returned to the caller once the API routine returns.  This makes it difficult, if not impossible (at least, as far as I am aware at this time), to distinguish between the case where a file was not provided, or it has the wrong permissions, or it is too small, or the block size while opening is not the same as that used when created ... all of these conditions simply return an error with error output "Invalid argument".  For example, one of the conditions you wanted to see tested was the "wrong bsize" code path, which I have covered.  If you dig deep enough into some of the other log files produced when the unit test runs (e.g. pmemblk3.log), you will see the "wrong bsize" line near the end of the file.  But by the time the error propagates back up to the caller, the error output is simply "Invalid argument", as is also the case with other types of errors.